src/infra/service/reducer.js

```diff
@@ -26,10 +27,29 @@ const group = (state = {
   features: [],
   selected: {},
   feeds: [],
+  comments: {},
 }, action) => {
   switch (action.type) {
     case 'GROUP_SET':
       return { ...state, ...action.data };
+    case 'GROUP_COMMENT_SET':
```

Comment:
expose this on `infra/repo/group` instead. So reducer will be clean. I prefer clean reducer, all business related logic that need storage goes to `repository`.